libs/core/tests/unit_tests/runnables/test_runnable.py

```diff
@@ -346,296 +346,8 @@ async def typed_async_lambda_impl(x: str) -> int:
     assert chat_prompt.input_schema.schema() == {
         "title": "PromptInput",
         "type": "object",
-        "properties": {
```

Comment:
Why did this change?

Reply:
Because of the change `get_input_schema`.

Reply:
I meant more of does the change look correct. In this case it looks like the change is introducing an unintentional bug? (It's usually a bad sign to see such a big change in a unit test.)

get_input_schema() is expected to provideda precise JSON schema description of what the variable `history` is (it's a list of messages for a chat prompt template). After this PR, this looks like it's no longer true -- which means that the change introduces a new bug

Reply:
Oh yes I incidently removed the `input_types` update in root_validator for `ChatPromptTemplate` which was causing this. But now it is fixed.